### PR TITLE
Feat: Added Snack-bar

### PR DESCRIPTION
**Pull request by abhijeetekad**
Added Snack-bar 
1)Base line snackbar
2)Leading snackbar
3)Stacked snack bar
chrome-capture (7)

### CODE REVIEW

File: components/snackbar.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+.snackbar {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 2rem 0rem;
```
Comment (prachi-sahani) [inline components/snackbar.css:4]: you need not mention unit for 0 margin or padding(you can use "margin: 2rem 0;")

File: documentation/doc.html [hunk 2]
```diff
@@ -585,8 +586,48 @@ <h4>The alert offers four severity levels that set a distinctive icon and color.
...
+          <h1 id="snackbar">SnackBar</h1>
+          <p>Snackbars provide brief notifications. The component is also known as a toast.</p>
+          <p>Click here 👇 to see live demo modal.</p>
+          <div class="snackbar">
+              <button class="base-line-btn normal-btn btn-login" id="base-line-btn">Base line</button>
+              <button class="leading-line-btn normal-btn btn-login" id="leading-line-btn">Leading</button>
+              <button class="stacked-line-btn normal-btn btn-login" id="stacked-line-btn">Stacked</button>
+          </div>
+          <div class="base-line-snackbar">
+            <section class="snackbar2">
```
Comment (prachi-sahani) [inline documentation/doc.html:598]: try using better class name(which is self-explanatory)

**Review by prachi-sahani — COMMENTED** [review]
code looks fine
suggestions:
- consider giving some gap between the action text(retry/undo) and action icon(cross icon)
- on hover of button the text is not visible. consider changing that style as well

File: components/snackbar.css (hunk 1 above)
Comment (abhijeetekad) [reply components/snackbar.css:4]: Ok.

File: components/snackbar.css [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+.base-line-snackbar {
+  position: fixed;
+  bottom: 1rem;
+  display: flex;
+  flex-direction: column;
```
Comment (rittik-ghosh) [inline components/snackbar.css:10]: You can use a utility class for the properties that are used often

File: components/snackbar.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  visibility: hidden;
+}
+.snackbar2 {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```
Comment (rittik-ghosh) [inline components/snackbar.css:16]: can use utility class

File: components/snackbar.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  box-shadow: 1.5px 1.5px var(--primary-color);
+}
+
+.leading-snackbar {
+  position: fixed;
+  bottom: 1.5rem;
+  left: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  visibility: hidden;
```
Comment (rittik-ghosh) [inline components/snackbar.css:31]: same here, utility class can be used

File: components/snackbar.css [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+/**********************/
+
+.stacked-snackbar {
+  position: fixed;
+  bottom: 1.5rem;
+  right: 1.5rem;
+  display: flex;
+  flex-direction: column;
```
Comment (rittik-ghosh) [inline components/snackbar.css:41]: same here

**Review by rittik-ghosh — COMMENTED** [review]
Apart from the changes suggested by prachi.
You can put the common code in the utility class so that the code look clean an readable.

File: documentation/doc.html (hunk 2 above)
Comment (abhijeetekad) [reply documentation/doc.html:598]: Done.

**Pull request merged**